client-course-schedulizer/src/utilities/hooks/useLocal.ts

```diff
@@ -7,7 +7,8 @@
  */
 export const loadLocal = <T>(key: string) => {
   const item: string | null = localStorage.getItem(key);
-  return item ? (JSON.parse(item) as T) : undefined;
+  const thing = item ? JSON.parse(item) : undefined;
+  return thing ? (thing as T) : undefined;
```

Comment:
Is there a reason why there is the intermediary step with `thing`?

```suggestion
  return item ? (JSON.parse(item) as T) : undefined;
```

Reply:
Oops, I was trying to return it to how it was before, but I forgot about that, thanks for the suggestion!